packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/element.js

```diff
@@ -88,9 +88,11 @@ export function build_set_attributes(
 		element_id,
 		is_dynamic ? attributes_id : b.literal(null),
 		b.object(values),
-		element.metadata.scoped &&
-			context.state.analysis.css.hash !== '' &&
-			b.literal(context.state.analysis.css.hash),
+		element.metadata.scoped && context.state.analysis.css.hash !== ''
+			? b.literal(context.state.analysis.css.hash)
+			: is_custom_element
+				? b.null
+				: false,
```

Comment:
what's this change about? Why `null` in one case and `false` in the other?

Reply:
It's just to generate a `null` instead of `undefined` for custom-element :
```js
$.set_attributes(cust_elem, attributes_1, { ...rest }, undefined, true, true);
// =>
$.set_attributes(cust_elem, attributes_1, { ...rest }, null, true, true);
```

But maybe it's useless...